docs/blueprints/linux-setup.md

```diff
@@ -0,0 +1,105 @@
+# Storage node installation and update on Linux
+
+## Abstract
+
+The design document outlines the steps needed to ease installation and provide auto-update of storage nodes on Linux without the use of Docker.
+
+
+## Background
+
+
+The idea is two have two main components, like for windows: the storage node binary and a storage node updater that will take care of updating the storage node binary according to the rollout version mechanism.
```

Comment:
Typo: "two have two" should be "to have two".

"windows" should be capitalized, i.e. "Windows".